src/main/java/com/weiliang/task/Event.java

```diff
@@ -0,0 +1,26 @@
+package com.weiliang.task;
+
+import java.time.LocalDateTime;
+import java.time.format.DateTimeFormatter;
+
+public class Event extends Task {
+
+    private LocalDateTime timing;
+
+    public Event(String task, String timing) {
+        super(task);
+        this.timing = LocalDateTime.parse(timing);
+    }
+
+    @Override
+    public String toFormattedString() {
+        return "E | " + (completed ? 1 : 0) + " | " + task + " | " + timing;
+    }
+
+    @Override
+    public String toString() {
+        return "[E][" + (completed ? "X" : " ") + "] " + task + " (at: "
+                + timing.format(DateTimeFormatter.ofPattern("dd MMMM YYYY, hh:mma")) + ")";
+    }
+
+}
```

Comment:
Similar to Deadline Class. The code follows Java Coding Standards and  a suggestion would be to add header comments to methods being override.